charts/consul/Chart.yaml

```diff
@@ -22,7 +22,7 @@ annotations:
     - name: consul-dataplane
       image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.3-dev
     - name: envoy
-      image: envoyproxy/envoy:v1.26.2
```

Comment:
Is there a reason why we downgraded from 1.26.2 to 1.25.11?

Reply:
Ah I see this was an inaccurate value that needed to be updated, please ignore.

Reply:
Yup. x-post for posterity https://github.com/hashicorp/consul-k8s/pull/3116#discussion_r1368872800